server/src/storage/datastore.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Datastore} from '@google-cloud/datastore';
+
+const datastore = new Datastore();
+
+const getWithId = async (kind: string, id: string) => {
```

Comment:
can => be removed?

Reply:
What do you mean by this? Use normal functions instead?

Reply:
Resolving, had a chat on Chat!